cmake/warnings.cmake

```diff
@@ -145,11 +145,14 @@ elseif (COMPILER_GCC)
     # Warn about anything that depends on the “size of” a function type or of void
     add_cxx_compile_options(-Wpointer-arith)
     # Warn if anything is declared more than once in the same scope
-    add_cxx_compile_options(-Wredundant-decls)
+    # add_cxx_compile_options(-Wredundant-decls)
```

Comment:
@zhanghuajieHIT better to fix warnings without disabling them.

Reply:
done